### ISSUE TITLE
Double clicking on result doesn't make tab permanent

### ISSUE THREAD

**Issue by adambuczynski** [atom/find-and-replace#680]
Like for double clicking in tree view, I'd like double clicking on a search result to make that tab permanent, not a preview tab.

Use case: often I want to find something in multiple files and open all the files that the search term was found in.

**Comment by swrobel** [comment]
I filed a similar request, but different: https://github.com/atom/find-and-replace/issues/694

**Comment by MadLittleMods** [comment]
Does this test still pass? https://github.com/atom/find-and-replace/blob/66f09c532bb4f7b941282b99d4daf85a08d2288c/spec/results-view-spec.coffee#L436

The double-click functionality was mentioned in the PR that changed the behavior, https://github.com/atom/find-and-replace/pull/644
> Double click opens file in regular non-pending state.

**Comment by swrobel** [comment]
@MadLittleMods not working for me either. Atom 1.7.0-beta4, MacOS.

**Comment by MadLittleMods** [comment]
Just noticed that if you double-click the tab in the preview/pending state, it will move to a permanent tab. Not the ideal workflow but workable for now.

**Comment by adambuczynski** [comment]
This seems to be fixed now (at least in 1.10.0-dev-672cf6d)

**Comment by adambuczynski** [comment]
I find this not working again in 1.10.0-dev-748dbcd, so it might be a bug that it sometimes works and sometimes doesn't.

**Comment by adambuczynski** [comment]
@mnquintana this issue seems to be a bug rather than an enhancement, could you update the label please?

**Comment by adambuczynski** [comment]
@50Wliu sorry, including you here as @mnquintana seems to be away for a while. Could you mark this as a bug instead? See https://github.com/atom/find-and-replace/pull/644, double click should open the pane in non-pending state.

Currently, with Atom 1.10.0 it's not working a lot of times, and double clicking the find result results in a temporary tab.

**Comment by adamreisnz** [comment]
Still not functional in 1.13 unfortunately. It's creating a lot of unnecessary effort when refactoring and needing to open a lot of files based on search results, as you need to then double click each tab to make it permanent.

**Comment by 50Wliu** [comment]
Yeah, I was never able to get this to work.

**Comment by adamreisnz** [comment]
Hasn't the mechanism for creating either temporary or permanent panes been refactored quite a while ago to allow for an easy API to choose how a file should be opened? I seem to recall reading a proposal for that way back when all new files were being opened in a temporary pane.

**Comment by TheNotary** [comment]
Changes like these should probably be looked at as feature deprecations if it can be foreseen that it will negatively impact a developer's workflow.  E.G. if she or he is relying on `exclusive use of permanent tabs` to open files, a warning should be listed that this behavior is going to change during an upcoming release and they he or she should begin to adjust their workflow or decline the upgrade.  

As a note, if double clicking the tab isn't working for people in certain releases, you may try right clicking and looking for a "make permanent tab" option.  

Does anyone know which version of atom killed the exclusive permanent tabs feature?  And is it still not implemented as a configuration option?  I hate to say it, but I think it may be a more efficient approach to editing text files than what has most recently been proposed and introduced into the stable branch of Atom.

**Comment by 50Wliu** [comment]
The config option you're looking for is "Allow Pending Pane Items".

**Comment by chrishiestand** [comment]
Just to make it easy "Allow Pending Pane Items" is a core configuration item, not a config option for "find and replace" http://flight-manual.atom.io/using-atom/sections/pending-pane-items/

**Comment by jcalfee** [comment]
Is there some place where tricks like this are documentation?  I end up upgrading then trying to figure out things like this is hit or miss.  If we at least had tips and tricks that would help out a lot..

**Comment by 50Wliu** [comment]
This only happens if the results view is not in a different pane from where the results are going to open.  Once the first click is registered, the new tab is opened and the second click doesn't register.

**Comment by jneuendorf** [comment]
The bug still happens on Atom 1.40.1 for me 😢 
Is this issue supposed to be working by now?

**Comment by jamesbebbington** [comment]
Yeah same for me in 1.40.1 with one pane. It's _really_ frustrating.

**Comment by gilbertohasnofb** [comment]
Still present in 1.46.0. I think this is a substantial inconsistency with how the users are expected to interact with the application, considering that the tree view allows for double clicks for opening permanent tabs. Opening individual search results and having to double click on their tab names is very tedious.
